themes/default/content/docs/guides/pulumi-packages/schema.md

```diff
@@ -0,0 +1,321 @@
+| `compatibility`        | `string`      | No       | Toggle compatibility mode for a specified target.                                                                        |
+| `usesIOClasses`        | `boolean`     | No       | Deprecated: This bool is no longer needed since all providers now use input/output classes.                              |
+| `emitPulumiPluginFile` | `boolean`     | No       | Indicates whether the pulumiplugin.json file should be generated.                                                        |
+
+For `go`:
+
+| Property                         | Type          | Required | Description                                                                                                                                              |
+|----------------------------------|---------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `importBasePath`                 | `string`      | No       | Base path for package imports (e.g. `github.com/pulumi/pulumi-kubernetes/sdk/go/kubernetes`).                                                            |
+| `rootPackageName`                | `string`      | No       | Explicit package name, which may be different to the import path.                                                                                        |
+| `moduleToPackage`                | `map[string]` | No       | Map from module -> package name (e.g. `{ "flowcontrol.apiserver.k8s.io/v1alpha1": "flowcontrol/v1alpha1" }`).                                            |
+| `packageImportAliases`           | `map[string]` | No       | Map from package name -> package alias (e.g. `{ "github.com/pulumi/pulumi-kubernetes/sdk/go/kubernetes/flowcontrol/v1alpha1": "flowcontrolv1alpha1" }`). |
+| `generateResourceContainerTypes` | `boolean`     | No       | Generate container types (arrays, maps, pointer output types etc.) for each resource. These are typically used to support external references.           |
+
+For `csharp`:
+
+| Property                 | Type          | Required | Description |
+|--------------------------|---------------|----------|-------------|
+| `packageReferences`      | `map[string]` | No       |             |
+| `namespaces`             | `map[string]` | No       |             |
+| `compatibility`          | `string`      | No       |             |
+| `dictionaryConstructors` | `boolean`     | No       |             |
+
+### PropertyLanguage
+
+Language-specific info for a property.
+
+For `python`:
+
+| Property  | Type      | Required | Description                                          |
+|-----------|-----------|----------|------------------------------------------------------|
+| `mapCase` | `boolean` | No       | Whether to use case mapping tables for the property. |
```

Comment:
This is dead code that I've been meaning to clean up. I don't think we need to document it.